devlab/tests/test_resource_migration.py

```diff
@@ -120,6 +120,39 @@ def test_migrate_nova_security_groups(self):
                                                 resource_name='security_group',
                                                 parameter='description')
 
+    def test_image_members(self):
+
+        def get_image_name(client, mem_id):
+            tenants = client.tenants.list()
+            return [t for t in tenants if t.id == mem_id][0].name
+
+        def member_list_collector(_images, client, auth_client):
+            _members = []
+            for img in _images:
+                members = client.image_members.list(img['id'])
+                if len(members) > 0:
+                    mbr_list = []
+                    for mem in members:
+                        mem_id = mem.__dict__['_info']['member_id']
+                        mem_name = get_image_name(auth_client, mem_id)
+                        mbr_list.append(mem_name)
+                    _members.append({img['name']: mbr_list})
+            return _members
+
+        src_images = self.src_cloud.glanceclient.images.list()
+        dst_images_gen = self.dst_cloud.glanceclient.images.list()
+        dst_images = [x.__dict__ for x in dst_images_gen]
+        filtering_data = self.filtering_utils.filter_images(src_images)
+        src_images = filtering_data[0]
+
+        src_members = member_list_collector(src_images,
+                                            self.src_cloud.glanceclient,
+                                            self.src_cloud.keystoneclient)
+        dst_members = member_list_collector(dst_images,
+                                            self.dst_cloud.glanceclient,
+                                            self.dst_cloud.keystoneclient)
+        self.assertEqual(src_members, dst_members)
```

Comment:
Think on modifying this to correctly compare lists, I'm not sure `assertEqual` compares lists correctly. For example:
```
for m in src_members:
    self.assertIn(m, dst_members)
```